packages/tall-plugin-meta-refresh/src/index.test.ts

```diff
@@ -0,0 +1,84 @@
+import { Readable } from 'node:stream'
+import { IncomingMessage } from 'node:http'
+import nock from 'nock'
+import { locationHeaderPlugin, tall, Follow, Stop } from '../../tall'
+import { metaRefreshPlugin } from './'
+
+beforeEach(() => {
+  nock.cleanAll()
+})
+
+test('it follows meta refresh redirects', async () => {
+  nock('https://example.com')
+    .get('/a-link')
+    .times(1)
+    .reply(
+      200,
+      `<html>
+         <head>
+           <meta
+             http-equiv="refresh"
+             content="0;url=https://dest.pizza/a-link"
+           />
+         </head>
+       </html>`
+    )
+  nock('https://dest.pizza').get('/a-link').times(1).reply(200, 'OK')
+
+  const url = await tall('https://example.com/a-link', {
+    plugins: [locationHeaderPlugin, metaRefreshPlugin]
+  })
+
+  expect(url).toBe('https://dest.pizza/a-link')
+})
+
+test('it does not follow meta refresh redirects if not found', async () => {
+  nock('https://example.com')
+    .get('/a-link')
+    .times(1)
+    .reply(
+      200,
+      `<html>
+         <head>
+         </head>
+       </html>`
+    )
+  nock('https://dest.pizza').get('/a-link').times(1).reply(200, 'OK')
+
+  const url = await tall('https://example.com/a-link', {
+    plugins: [locationHeaderPlugin, metaRefreshPlugin]
+  })
+
+  expect(url).toBe('https://example.com/a-link')
+})
+
+test('it stop parsing as soon as the meta redirect tag is found', async () => {
+  const response = Readable.from([
+    `<html>
+  <head>
+  <meta
+    http`,
+    `-equiv="refresh"
+    content="0;url=https://dest.pizza/a-link"
+  />`,
+    `</head>
+  <body>
+   <h1>hello</h1>
+   <p>Hello World</p>
+ </body>
+</html>`
```

Comment:
lining up the tags vertically

```suggestion
       <head>
         <meta
           http`,
    `        -equiv="refresh"
           content="0;url=https://dest.pizza/a-link"
         />`,
    `  </head>
       <body>
         <h1>hello</h1>
         <p>Hello World</p>
       </body>
     </html>`
```